develop/index.md

```diff
@@ -6,29 +6,193 @@ hide:
+## Introduction
 
-This section builds on concepts from the [Polkadot Protocol](/polkadot-protocol){target=\_blank} section, providing the practical knowledge and resources needed to design, deploy, and operate scalable solutions in the Polkadot ecosystem. By leveraging Polkadot's unique features, such as interoperability and decentralized security, developers can build applications and infrastructure tailored to their needs.
+This guide is a starting point for developers who wish to build in the Polkadot ecosystem. To get the most from this section: 
 
-## Choose Your Development Path
+1. Identify your development pathway:
+    - [**Parachain Developers**](#parachain-developers) - build, deploy, and maintain custom parachains with the Polkadot SDK
+    - [**Smart Contract Developers**](#smart-contract-developers) - leverage smart contracts and execute custom logic over existing chains to streamline your development process
+    - [**Application Developers**](#application-developers) - leverage Polkadot's underlying protocol features to create solutions for your users to interact with the ecosystem
+2. Use the sections under your pathway as follows:
+    - **Learn** - links connecting you to content to deepen your knowledge and understanding 
+    - **Build** - links connecting you to content to help you build, including goal-oriented guides and step-by-step tutorials
+    - **Tools** - links connecting you to tools commonly used in your pathway
+    - **Resources** - links connecting you to resources for your pathway, including references, code repositories, and outside documentation
 
-- **Parachain developers** - build, deploy, and maintain custom parachains with the Polkadot SDK. From runtime customization to network operations, learn how to create a scalable and efficient parachain
+## Development Pathways
 
-    - **Where to start** - [Introduction to the Polkadot SDK](/develop/parachains/get-started/intro-polkadot-sdk/){target=\_blank}
+Developers can choose from different development pathways to build applications and core blockchain functionality. Each pathway caters to different types of projects and developer skill sets while complementing one another within the broader network.
 
-- **Smart contract developers** - utilize Polkadot's support for Wasm-based contracts with ink! or deploy Solidity contracts on EVM-compatible parachains. Leverage familiar tools to write, test, and manage decentralized logic
+The Polkadot ecosystem provides multiple development pathways:
 
-    - **Where to start** - [Overview of the Smart Contract Landscape on Polkadot](/develop/smart-contracts/overview/){target=\_blank}
+```mermaid
+graph TD
+    A[Development Pathways]
+    A --> B[Parachain Development]
+    A --> C[Smart Contract Development]
+    A --> D[Client-Side Development]
```

Comment:
Let's use one word for this everywhere 

Which one? 

1. Client Side
2. Application
3. Offchain Application


I personally like the last one, assuming it is clearly explained (added to the glossary)

Both Parachain, smart contract (and solochain) are on-chain applications.

Everything else is offchain, uncluding whatever you build with PJS, PAPI, SubXT.

I'd leave it to you to decide, but please make it consistent

If using offchain/onchain terms, please add both and link to glossary. Import to define such things well.

Reply:
Might need refactoring in other pages too!

Reply:
Thanks, Kian! You are right. Consistency is important. Explaining off-chain/on-chain as part of finding the correct dev path might not be the best dev ex. "Application" feels better to me. I will communicate with our folks and scan the docs to update as needed.